libmamba/include/mamba/specs/archive.hpp

```diff
@@ -15,10 +15,9 @@
 
 namespace mamba::specs
 {
-    inline static constexpr auto ARCHIVE_EXTENSIONS = std::array{
-        std::string_view(".tar.bz2"),
-        std::string_view(".conda"),
-    };
+    inline static constexpr auto ARCHIVE_EXTENSIONS = std::array{ std::string_view(".tar.bz2"),
```

Comment:
Should we split that in two? Or even just hard code it where we need it?
I fear adding pypi code in Conda abstactions will just make it harder on the long run.